apps/pg_subscriptor/example/init.sql

```diff
@@ -7,3 +7,5 @@ CREATE TABLE distributors (
      did    integer NOT NULL,
      name   varchar(255) NOT NULL
 );
+
+CREATE PUBLICATION postgrex_example FOR ALL TABLES;
```

Comment:
So this triggers RELATION message, which we need for update (and maybe more)?
Maybe add a comment describing it to this?